### ISSUE TITLE
[bug]: BreadcrumbEllipsis accessible name

### ISSUE THREAD

**Issue by dwightjack** [shadcn-ui/ui#8074]
### Describe the bug

In the v4 [`BreadcrumbEllipsis`](https://github.com/seongminn/shadcn-ui/blob/c41c6ece86b5de472668729ed336ee4aac96ee16/apps/v4/registry/new-york-v4/ui/breadcrumb.tsx#L88-L97) component I noticed a possible accessibility inconsistency:

```jsx
<span
  data-slot="breadcrumb-ellipsis"
  role="presentation"
  aria-hidden="true"
  className={cn("flex size-9 items-center justify-center", className)}
  {...props}
>
  <MoreHorizontal className="size-4" />
  <span className="sr-only">More</span>
</span>
```

Since the root `span` is hidden via `aria-hidden`, the node including the `More` text is ignored. 

### How to fix

I think that the issue could be fixed by just removing the `aria-hidden` attribute (and maybe adding it to the `svg`, just in case).  
  

### Affected component/components

Breadcrumb

### How to reproduce

For example, inspecting the [_Collapsed_](https://ui.shadcn.com/docs/components/breadcrumb#collapsed) example with Chrome accessibility tree I see that the node doesn't have a computed accessible name.

<p>

<img width="425" height="129" alt="Image" src="https://github.com/user-attachments/assets/a5ac0184-77a7-4824-80bb-6864e36cbcb1" />

</p>
<p>

<img width="357" height="266" alt="Image" src="https://github.com/user-attachments/assets/cf9ae62a-46bb-46d5-be0e-236aa8a27a65" />

</p>

### Logs

```bash

```

### System Info

```bash
* Shadcn UI, latest v4 version
* React 19.1.1
```

### Before submitting

- [x] I've made research efforts and searched the documentation
- [x] I've searched for existing issues
